[IOTDB-127] Fix documents problems

samperson1997 opened the pull request:
Fix some documents problems in both Chinese and English documents as is mentioned in issue [IOTDB-127](https://issues.apache.org/jira/browse/IOTDB-127).
The website links are also changed in [this PR](https://github.com/apache/incubator-iotdb-website/pull/25) to this one.

jt2594838 commented on docs/Documentation/UserGuideV0.7.0/2-Concept Key Concepts and Terminology/1-Key Concepts and Terminology.md:
```diff
@@ -95,12 +95,12 @@ When `*` appears in the middle of the path, it represents `*` itself, i.e., a la
 * Timestamp
 
-The timestamp is the time point at which a data arrives. IoTDB timestamps are divided into two types: LONG and DATETIME (including DATETIME-INPUT and DATETIME-DISPLAY). When a user enters a timestamp, he can use a LONG type timestamp or a DATETIME-INPUT type timestamp, where the support format of the DATETIME-INPUT type timestamp is shown in Table 2-1.
+The timestamp is the time point at which a data arrives. IoTDB timestamps are divided into two types: LONG and DATETIME (including DATETIME-INPUT and DATETIME-DISPLAY). When a user enters a timestamp, he can use a LONG type timestamp or a DATETIME-INPUT type timestamp, where the support format of the DATETIME-INPUT type timestamp is shown in table:
```
I suggest "at which data is produced" over "at which a data arrives" because data arrival time for a database is just another thing.
"user enters" -> "user inputs"
"where the support format of the DATETIME-INPUT type timestamp is shown in table" -> "and the supported formats of the DATETIME-INPUT type timestamp are shown in the table below"

jt2594838 commented on docs/Documentation/UserGuideV0.7.0/2-Concept Key Concepts and Terminology/1-Key Concepts and Terminology.md:
```diff
@@ -95,12 +95,12 @@ When `*` appears in the middle of the path, it represents `*` itself, i.e., a la
+The timestamp is the time point at which a data arrives. IoTDB timestamps are divided into two types: LONG and DATETIME (including DATETIME-INPUT and DATETIME-DISPLAY). When a user enters a timestamp, he can use a LONG type timestamp or a DATETIME-INPUT type timestamp, where the support format of the DATETIME-INPUT type timestamp is shown in table:
 
-<center>**Table 2-1 Support format of DATETIME-INPUT type timestamp**
+<center>**Support format of DATETIME-INPUT type timestamp**
```
"Support format" -> "Supported formats".

jt2594838 commented on docs/Documentation/UserGuideV0.7.0/2-Concept Key Concepts and Terminology/1-Key Concepts and Terminology.md:
```diff
@@ -129,9 +129,9 @@ The timestamp is the time point at which a data arrives. IoTDB timestamps are di
 </center>
 
-IoTDB can support LONG types and DATETIME-DISPLAY types when displaying timestamps. The DATETIME-DISPLAY type can support user-defined time formats. The syntax of the custom time format is shown in Table 2-2.
+IoTDB can support LONG types and DATETIME-DISPLAY types when displaying timestamps. The DATETIME-DISPLAY type can support user-defined time formats. The syntax of the custom time format is shown in table:
```
"table" -> "the table below"

jt2594838 commented on docs/Documentation/UserGuideV0.7.0/2-Concept Key Concepts and Terminology/2-Data Type.md:
```diff
@@ -31,7 +31,7 @@ IoTDB supports six data types in total:
 
 
-The time series of **FLOAT** and **DOUBLE** type can specify (MAX\_POINT\_NUMBER, see [this page](/#/Documents/latest/chap5/sec1) for more information on how to specify), which is the number of digits after the decimal point of the floating point number, if the encoding method is [RLE](/#/Documents/latest/chap2/sec3) or [TS\_2DIFF](/#/Documents/latest/chap2/sec3) (Refer to [Create Timeseries Stat ...
+The time series of **FLOAT** and **DOUBLE** type can specify (MAX\_POINT\_NUMBER, see [this page](/#/Documents/latest/chap5/sec1) for more information on how to specify), which is the number of digits after the decimal point of the floating point number, if the encoding method is [RLE](/#/Documents/latest/chap2/sec3) or [TS\_2DIFF](/#/Documents/latest/chap2/sec3) (Refer to [Create Timeseries Stat ...
```
remove wordy "for configuration for the configuration method".

jt2594838 commented on docs/Documentation/UserGuideV0.7.0/3-Operation Manual/2-Data Model Selection.md:
````diff
@@ -99,12 +99,7 @@ The results are shown below respectly:
-IoTDB> show timeseries root.ln.wf03
-Msg: Failed to fetch timeseries root.ln.wf03's metadata because: Timeseries does not exist.
-```
+It is worth noting that when the path queries does not exist, the system will return no timeseries.  
````
"the path queries" -> "the queried path"

jt2594838 commented on docs/Documentation/UserGuideV0.7.0/3-Operation Manual/3-Data Import.md:
````diff
@@ -43,7 +43,7 @@ IoTDB > insert into root.ln.wf02.wt02(timestamp,status) values(1,true)
 ```
 
-The above example code inserts the long integer timestamp and the value "true" into the timeseries `root.ln.wf02.wt02.status` and inserts the long integer timestamp and the value "v1" into the timeseries `root.ln.wf02.wt02.hardware`. When the execution is successful, a prompt "execute successfully" will appear to indicate that the data insertion has been completed.
+The above example code inserts the long integer timestamp and the value "true" into the timeseries `root.ln.wf02.wt02.status` and inserts the long integer timestamp and the value "v1" into the timeseries `root.ln.wf02.wt02.hardware`. When the execution is successful, cost time will appear to indicate that the data insertion has been completed.
````
"cost time will appear" -> "cost time is shown"

samperson1997 commented:
> It seems that you do not only correct the Chinese version but also the English version, so you'd better use a more precise title of the PR.

Many thanks for the review and suggestions! I have changed the title and fixed the detailed problems.
